frontend/src/component/integrations/IntegrationForm/IntegrationForm.tsx

```diff
@@ -71,10 +71,12 @@ export const IntegrationForm: VFC<IntegrationFormProps> = ({
         value: environment.name,
         label: environment.name,
     }));
-    const selectableEvents = provider?.events?.map(event => ({
-        value: event,
-        label: event,
-    }));
+    const selectableEvents = provider?.events
+        ?.map(event => ({
+            value: event,
+            label: event,
+        }))
+        .sort((a, b) => a.label.localeCompare(b.label));
```

Comment:
We now have a bunch of events, so it's better if we sort them. Ideally they would be grouped, but maybe that can come in a later PR.